regions_www/tabs/manual_body.html

```diff
@@ -157,6 +157,18 @@ <h4><a name="faqs"></a>Additional FAQs</h4>
                 </table>
               </div>
             </div>
+            <p><a href="javascript:toggleElement(&#39;a4&#39;)">4. What if I'm interested in trips not people? And does PCT account for return trips?</a></p>
```

Comment:
Wow, is that how we do things in the manual? Hope it wasn't hard to work out, `href="javascript:toggleElement(&#39;a4&#39;)"` looks a touch esoteric.

Reply:
Agreed esoteric-looking, I just copied it from above...

Reply:
Took me bit to see what it did, I guess it was auto-generated from Rmarkdown once.  There are many ways to improve to make it simpler and easier to understand.  The standard way to use [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes), then write similar Javascript code to open and close based on the attributes BUT I'm not sure it is worth the effort for the manual and it does work :smile: